src/pks/mpc/mpc_permafrost_split_flux_columns.cc

```diff
@@ -79,10 +79,10 @@ MPCPermafrostSplitFluxColumns::MPCPermafrostSplitFluxColumns(Teuchos::ParameterL
     cv_key_ = Keys::readKey(*plist_, domain_star, "cell volume", "cell_volume");
   
     // set up for a primary variable field evaluator for the flux
-    Key primary_pkey = Keys::getKey(domain_surf + std::string("_*"), p_lateral_flow_source_suffix_);
+    Key primary_pkey = Keys::getKey(domain_surf + std::string(":*"), p_lateral_flow_source_suffix_);
```

Comment:
These can be:  `Key primary_pkey = Keys::getKey(domain_surf, "*", p_lateral_flow_source_suffix_);`

I added a bunch of these kind of helpers at: https://github.com/amanzi/amanzi/blob/master/src/utils/Key.hh#L90

Reply:
done